packages/components/src/components/link/_link.scss

```diff
@@ -91,6 +91,14 @@
   .#{$prefix}--link--disabled.#{$prefix}--link--inline {
     text-decoration: underline;
   }
+
+  .#{$prefix}--link--sm {
+    font-size: rem(12px);
```

Comment:
I think we'll want adjust line height/letter-spacing as well

Reply:
Hi @vpicone, the classes are now using the tokens you mentioned

Reply:
Awesome! We’ll just wait for design verification to make sure those are the right tokens.

Reply:
`helper-text-01` isn't actually the correct token. Its more helper text that is associated with fields. We don't really have a token that matches this 12px usage right now. Does it absolutely have to have a token? If so I think we may need to make a new type token that supports it. 

@jeanservaas what do you think?

Reply:
@aagonzales it doesn't have to have a token, we can explicitly define each attribute. But either way, I kind of assumed we'd want to match the line-height and letter-spacing of similarly sized text.

Reply:
Yup we would want to match those other attributes. I talked to Jeannie about this and we want to add a new type token for the 12px scenario which has been coming up more. `helper-text-01` is actually going to be deprecated in the next release according to Jeannie.

Reply:
Hi both, I have a couple of questions regarding what you discussed above. 

If a new token is needed, is it something that has to be done as part of this PR? If no, I can for the time being just add the CSS needed to `--link--sm` and once the token is created it can replace the CSS in another issue perhaps. If yes, that's fine I can add the changes in another commit, assuming to create a new one I just need to use eg `helper-text-01` or some other token as an example. Either way, it would be great if you let me know what the line-height and the letter-spacing need to be set to.

And finally, is `--link--lg` okay using `body-long-02` or would that need changing too?

Thanks!

Reply:
Yes the new token would be a task outside of this issue. So I guess its ok to proceed with using `help-text-01` for now and we can replace it later (unless dev disagrees with that @vpicone @tw15egan @emyarod )

And I was distracted by that token convo, but the `--lg` link should be using `body-short-02` to match the line height sequencing of reg link which uses `body-short-01`

Reply:
No worries @aagonzales! I'll wait until a decision's been made around `--link--sm`, and I'll change the other one at that point as well

Reply:
@rzhekkova I don’t think a link variant token makes sense, since it would only be used in this one place anyway. IMO using the helper-text token now will make it easier to find and clean up when we actually deprecate the token.

Reply:
It wouldn't be link specific. It would be a new general 12px type style. Jeannie has come across other places where she's need a 12px style that isn't a label or caption.

Reply:
Ok cool, that makes sense. So, in that case I'll just change `--link--lg` to use `body-short-02` as per Anna's message above

Reply:
@aagonzales for sure, that’s was my concern for a link—sm token suggested here. 

Using the existing helper token would make it easier to find and replace instances in the future when the general token is published. Easier than trying to find/replace chunks of css properties.

Reply:
Hi guys, I've updated the large link to use `body-short-02`. Let me know if any other changes are needed :)